woocommerce-gateway-mondido.php

```diff
@@ -31,6 +31,11 @@ public function __construct() {
 		register_deactivation_hook( __FILE__, array( $this, 'flush_rewrite_rules' ) );
 
 		// Actions
+		add_action( 'before_woocommerce_init', function() {
+			if ( class_exists( \Automattic\WooCommerce\Utilities\FeaturesUtil::class ) ) {
```

Comment:
I think this call should move to either `OrderStorageTechnology` or `OrderUtil` so that we have all mentions of the custom orders table in a single location. However, if we move the logic we must pass `__FILE__` as an argument to preserve the behavior, e.g. `OrderStorageTechnology::declare_compatibility(__FILE__)`;

Reply:
My idea was just to move the mention of the custom orders table to either `OrderStorage` or `OrderUtil` since that is an implementation detail of HPOS. Thinking of it now, an even better location is probably `HPOSOrderStorage`. The call to `add_action` is probably best kept here. 
```
add_action( 'before_woocommerce_init', function() {
	HPOSOrderStorage::declare_compatibility(__FILE__);
}
```
However, if you prefer the current solution I am fine with that.